src/autopredictor/show_all.py

```diff
@@ -0,0 +1,19 @@
+def show_all(X):
+    """
+    This function converts the trained regressor model scores stored in a dictionary by 
```

Comment:
Hi Sharon, do you think it should be regression?
```suggestion
    This function converts the trained regression model scores stored in a dictionary by 
```